README.md

```diff
@@ -1 +1,31 @@
-# animal-shelter
\ No newline at end of file
+# Animal shelter
```

Comment:
I am not sure about the aim of these three paragraphs "Motivation, Aim, The main goal" maybe we should consider adding content of these paragraphs under this header - something like what we do and why. What do you think?

Reply:
Corrected.